query/src/servers/mysql/mysql_interactive_worker.rs

```diff
@@ -251,70 +251,91 @@ impl<W: std::io::Write> InteractiveWorkerBase<W> {
 
     async fn do_close(&mut self, _: u32) {}
 
-    fn federated_server_setup_set_or_jdbc_command(&mut self, query: &str) -> bool {
-        let expr = RegexSet::new(&[
-            "(?i)^(SET NAMES(.*))",
-            "(?i)^(SET character_set_results(.*))",
-            "(?i)^(SET FOREIGN_KEY_CHECKS(.*))",
-            "(?i)^(SET AUTOCOMMIT(.*))",
-            "(?i)^(SET sql_mode(.*))",
-            "(?i)^(SET @@(.*))",
-            "(?i)^(SHOW VARIABLES(.*))",
-            "(?i)^(SET SESSION TRANSACTION ISOLATION LEVEL(.*))",
+    fn federated_server_setup_set_or_jdbc_command(&mut self, query: &str) -> Option<DataBlock> {
+        let rules: Vec<(&str, Option<DataBlock>)> = vec![
+            ("(?i)^(SELECT @@(.*))", None),
```

Comment:
`select @@tx_isolation` should returns `AUTOCOMMIT`:

/home/bohu/.local/lib/python3.6/site-packages/pandas/io/sql.py:1162: SAWarning: Could not retrieve transaction isolation level for MySQL connection.